.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,23 @@
+name: C/C++ CI
```

Comment:
Do you need this file? Is the Makefile missing?

Reply:
Please ignore any files in that workflow folder and is nothing to do with audiogridder code in any case. That file created when I was experimenting with github feature for CI/CD which gives few hours for free every month. Idea  is to make the the development process as per standards.

Reply:
Yeah I know gh workflows. I was just wondering if you checked it in by mistake, as the workflow uses make but you did not add a Makefile.